otelcollector/test/testkube/config-processing-test-crs/testkube-config-test-default-targets-on-v2-crs.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: tests.testkube.io/v3
+kind: Test
+metadata:
+  name: configprocessingcommonv2
+  namespace: testkube
+  labels:
+    executor: ginkgo-executor
+    test-type: ginkgo-test
+spec:
+  type: ginkgo/test
+  content:
+    type: git
+    repository:
+      type: git
+      uri: https://github.com/Azure/prometheus-collector
+      branch: main
+      path: otelcollector/test/ginkgo-e2e
+  executionRequest:
+    variables:
+      AMW_QUERY_ENDPOINT:
+        name: AMW_QUERY_ENDPOINT
+        value: "$AMW_QUERY_ENDPOINT"
+        type: basic
+      AZURE_CLIENT_ID:
+        name: AZURE_CLIENT_ID
+        value: "$AZURE_CLIENT_ID"
+        type: basic
+    args:
+      - "--label-filter"
+      - "(config-processing-common, config-processing-common-with-config-maps-v2)"
+      - "./configprocessing"
+    executePostRunScriptBeforeScraping: false
+---
+apiVersion: tests.testkube.io/v3
+kind: Test
+metadata:
+  name: defaultsettingsenabledv2
+  namespace: testkube
+  labels:
+    executor: ginkgo-executor
+    test-type: ginkgo-test
+spec:
+  type: ginkgo/test
+  content:
+    type: git
+    repository:
+      type: git
+      uri: https://github.com/Azure/prometheus-collector
+      branch: main
+      path: otelcollector/test/ginkgo-e2e
+  executionRequest:
+    variables:
+      AMW_QUERY_ENDPOINT:
+        name: AMW_QUERY_ENDPOINT
+        value: "$AMW_QUERY_ENDPOINT"
+        type: basic
+      AZURE_CLIENT_ID:
+        name: AZURE_CLIENT_ID
+        value: "$AZURE_CLIENT_ID"
+        type: basic
+    args:
+      - "--label-filter"
+      - "(config-processing-common-with-config-maps-v2)"
+      - "./configprocessing"
+    executePostRunScriptBeforeScraping: false
+---
+apiVersion: tests.testkube.io/v3
+kind: TestSuite
+metadata:
+  name: config-tests-def-targets-on-configmap-v2
```

Comment:
I think this new testsuite needs a new task in azure-pipeline-config-tests.yml to run it with the v2 configmaps applied on the cluster

Reply:
added